helix-term/src/commands.rs

```diff
@@ -870,7 +870,7 @@ fn goto_window(cx: &mut Context, align: Align) {
     let config = cx.editor.config();
     let (view, doc) = current!(cx.editor);
 
-    let height = view.inner_area().height as usize;
+    let height = view.inner_area(doc).height as usize;
```

Comment:
In almost every circumstance we're only interested in the height which is unaffected by gutters. Let's add a view.height() method that calculates it without having to pass in a doc. Additionally the method should return a usize since we're casting to usize everywhere

Reply:
Sounds good - I added a `view.inner_height()` which is just slightly different than a `view.height()` method because it subtracts the statusbar height allowing it to be dropped in for all the places where `inner_area().height` was being used previously.